test/integration/sidecar/sidecar_stack.go

```diff
@@ -0,0 +1,448 @@
+package sidecar
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	appmesh "github.com/aws/aws-app-mesh-controller-for-k8s/apis/appmesh/v1beta2"
+	"github.com/aws/aws-app-mesh-controller-for-k8s/test/framework"
+	"github.com/aws/aws-sdk-go/aws"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	defaultFrontendImage = "public.ecr.aws/b7m0w2t6/color-fe-app:2.0.3"
+	defaultBackendImage  = "public.ecr.aws/b7m0w2t6/color-be-app:2.0.2"
```

Comment:
these are the application images in the `/test_app/sidecar-backend` and `/test_app/sidecar-frontend`